README.rst

```diff
@@ -191,49 +189,32 @@ meta taking priority.
             #     'download_slot': 'quotes.toscrape.com'
             # }
 
-There is a shortcut, in case a request uses the same parameters as
-defined in the ``ZYTE_API_DEFAULT_PARAMS`` setting, without any further
-customization - the ``zyte_api`` meta key can be set to ``True`` or ``{}``:
+``ZYTE_API_DEFAULT_PARAMS`` does not make requests automatically go through
+Zyte Data API. See :ref:`enabled`.
 
-.. code-block:: python
+Parameters in ``ZYTE_API_DEFAULT_PARAMS`` are merged with parameters set via
+the ``zyte_api`` meta key, with the values in meta taking priority.
 
-    import scrapy
 
+.. _enabled:
 
-    class SampleQuotesSpider(scrapy.Spider):
-        name = "sample_quotes"
+Controlling which requests go through Zyte Data API
+---------------------------------------------------
 
-        custom_settings = {
-            "ZYTE_API_DEFAULT_PARAMS": {
-                "browserHtml": True,
-            }
-        }
+By default, only requests where the ``zyte_api`` key in Request.meta_ is set to
+``True`` or set to a dictionary go through Zyte Data API.
 
-        def start_requests(self):
-            yield scrapy.Request(
-                url="http://quotes.toscrape.com/",
-                callback=self.parse,
-                meta={"zyte_api": True},
-            )
+.. _Request.meta: https://docs.scrapy.org/en/latest/topics/request-response.html#scrapy.http.Request.meta
 
-        def parse(self, response):
-            yield {"URL": response.url, "HTML": response.body}
+Set the ``ZYTE_API_ALL`` setting to ``True`` to make all requests go through
```

Comment:
I think `ZYTE_API_ON_ALL_REQUESTS: bool` sounds more clear.

Let's also mention the default value here.